acestep/ui/aceflow/run.py

```
@@ -0,0 +1,19 @@
+import argparse
+
+import uvicorn
+
+from .app import create_app
+
+
+def main():
+    ap = argparse.ArgumentParser()
+    ap.add_argument("--host", default="0.0.0.0")
```

Ruff (0.15.4)

[error] Possible binding to all interfaces

(S104)